src/domain/users/index.types.d.ts

```diff
@@ -7,11 +7,29 @@ type UserLanguage =
 declare const deviceTokenSymbol: unique symbol
 type DeviceToken = number & { [deviceTokenSymbol]: never }
 
+type UserContact = {
+  id: string
+  name: string
+  transactionsCount: number
+}
+
+type QuizQuestion = {
+  id: string
+  earnAmount: number
+}
+
+type UserQuizQuestion = {
+  question: QuizQuestion
+  completed: boolean
+}
+
 type User = {
   id: UserId
   username: Username | null
   phone: PhoneNumber
   language: UserLanguage
+  contacts: UserContact[]
```

Comment:
I have a doubt here, if we have contacts at user level and not at account level.. will we show the transactions for all wallets in contact info? should that screen be redesigned?

Reply:
Not sure I understand. Contact does not expose transactions.